packages/rainbowkit/src/components/ConnectOptions/ConnectDetails.tsx

```diff
@@ -196,19 +200,21 @@ export function ConnectDetail({
     href?: string;
   } | null = showWalletConnectModal
     ? {
-        description: `Need the ${
-          compactModeEnabled ? '' : 'official'
-        } WalletConnect modal?`,
-        label: 'OPEN',
+        description: `${i18n.t('intro.wallet.get_started.need')} ${
+          compactModeEnabled ? '' : i18n.t('intro.wallet.get_started.official')
+        } ${i18n.t('intro.wallet.get_started.wallet_connect')}`,
+        label: i18n.t('intro.wallet.get_started.open'),
         onClick: () => {
           onClose();
           showWalletConnectModal();
         },
       }
     : hasQrCode
     ? {
-        description: `Don\u2019t have ${name}?`,
-        label: 'GET',
+        description: `${i18n.t('intro.wallet.get_started.not_found', {
```

Comment:
We should bring over `\u2019` to the string; I believe this was done to prevent a style system issue. There are a few other instances of this; shouldn't have a problem translating